src/main/java/net/skinsrestorer/bungee/SkinsRestorer.java

```diff
@@ -242,4 +244,9 @@ public void upToDate() {
             }
         }));
     }
+
+    @Override
+    public SRApplier getApplier() {
+        return skinApplierBungee;
+    }
```

Comment:
Should this not be a @getter ?

Reply:
i can't use @Getter because its part of the interface